glide-core/src/client/value_conversion.rs

```diff
@@ -575,6 +689,136 @@ pub(crate) fn expected_type_for_cmd(cmd: &Cmd) -> Option<ExpectedReturnType> {
 mod tests {
     use super::*;
 
+    #[test]
+    fn convert_function_list() {
+        let expected_type = ExpectedReturnType::ArrayOfMaps(&ExpectedReturnType::ArrayOfMaps(
```

Comment:
from Andrew's PR:
use the expected_type_for_cmd instead of hardcoding it here, e.g.:

        let expected_type = expected_type_for_cmd(cmd.arg("STREAMS").arg("key").arg("id"));